Update help_bits_learn.md (#34636)
DataDog/documentation: content/en/bits_ai/bits_ai_sre/knowledge_sources.md

You can either add step-by-step troubleshooting instructions directly in the monitor message or link to a Confluence page that contains those instructions.

> - **Include Datadog telemetry links**: When adding instructions in the monitor message, include links to the most relevant telemetry. Start with the first place you'd normally look in Datadog when the monitor triggers, for example, a dashboard, logs, traces, or a notebook with key widgets. Links don’t need special formatting; plain URLs work.

Reviewer: ```suggestion
- **Include Datadog telemetry links**: When adding instructions in the monitor message, include links to the most relevant telemetry. Start with the first place you'd normally look in Datadog when the monitor triggers, such as a dashboard, logs, traces, or a notebook with key widgets. Links don't need special formatting; plain URLs work.
```
